core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/action/ActionConfiguration.java

```diff
@@ -46,7 +46,8 @@ private ActionConfiguration() {
                 .getTableNameByLabel("Actions")
                 + " where ACTION_ID = " + SQLTools.getStringForSQL(actionKey.getActionId()) +
                 " AND SCRIPT_ID = " + SQLTools.getStringForSQL(actionKey.getScriptKey().getScriptId()) +
-                " AND SCRIPT_VRS_NB = " + SQLTools.getStringForSQL(actionKey.getScriptKey().getScriptVersion()) + ";";
+                " AND SCRIPT_VRS_NB = " + SQLTools.getStringForSQL(actionKey.getScriptKey().getScriptVersion()) +
+                " AND DELETED_AT = 'NA' ;";
```

Comment:
doesn't ActionKey contain deletedAt? don't we need to use this value?

Reply:
Yes ActionKey contains deletedAt but for non-deleted record we should explicitly write 'NA'.